plone/app/layout/globals/layout.py

```diff
@@ -115,7 +116,9 @@ def thumb_visible(self):
             return True
         else:
             return False
-
+    @deprecate('deprecated since Plone 4, ContentIcons are rendered \
+    as Fonts now see \
```

Comment:
@jensens this *now* is probably clear *now* (sorry about that), but not in half a year time, nor after Plone 5.2 , Plone 6 nor Plone 7. Would you mind changing that to *5.1/5.0* or so?